packages/db/tests/connection.js

```diff
@@ -0,0 +1,99 @@
+const test = require('ava');
+const sinon = require('sinon');
+const proxyquire = require('proxyquire').noPreserveCache();
+
+const sandbox = sinon.createSandbox();
```

Comment:
Do we need sinon here? Can we just use `() => fakeConnectionConfig`?

Reply:
Yes: 3fe643057